common/auth/tls_config_helper_test.go

```diff
@@ -37,12 +39,33 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const validBase64CaData = "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURhekNDQWxPZ0F3SUJBZ0lVS0R0SG5kbnlaSldQV21ONnBEYmY2aTZZYVlnd0RRWUpLb1pJaHZjTkFRRUwKQlFBd1JURUxNQWtHQTFVRUJoTUNRVlV4RXpBUkJnTlZCQWdNQ2xOdmJXVXRVM1JoZEdVeElUQWZCZ05WQkFvTQpHRWx1ZEdWeWJtVjBJRmRwWkdkcGRITWdVSFI1SUV4MFpEQWVGdzB5TXpFeE16QXdOVEEwTURsYUZ3MHlNekV5Ck16QXdOVEEwTURsYU1FVXhDekFKQmdOVkJBWVRBa0ZWTVJNd0VRWURWUVFJREFwVGIyMWxMVk4wWVhSbE1TRXcKSHdZRFZRUUtEQmhKYm5SbGNtNWxkQ0JYYVdSbmFYUnpJRkIwZVNCTWRHUXdnZ0VpTUEwR0NTcUdTSWIzRFFFQgpBUVVBQTRJQkR3QXdnZ0VLQW9JQkFRRFZkQ1FraGVHeU5LVnNib2xRMUFlK0xONmJ4eHNzQjMvcWNEWmdDclVlCkwxbEF6WHJYN1FJT2k4MUE5Nk5LUWQzOTZON3VRRE1jaWdDak1lS2pWNTRDQzNwejBXaVpmWVdHSzBiMXU3NHUKR1hhejdRa01DMXFyRWpSWjhoYml3M1orbGx1TGxlQXkzdkpNUmt3OEdqcHF4eVdrNFZNd2lzR1MzVXZBV0pqSgo3M0VRa0UyL09Ia05tbXEwTWlFcU5JVWdUZFNCR3dqSWh1UmtwVDlUQkhWNTNvcDduTjJzaHgzMUZWalIycEdsCkY0b05WOGl0RVNSVVJzOHRDaHJNWlcyQ0JLN1lSRzRKbVBKL2NYQ3grMW1aTmJId2JDOWdySVdRQis5bUNUZC8KYXV6VW85Y2hXdy9xNjROV1BOQnZsdnV1SFhNUVkyZkhkaU5pVG1XbHdLMHpBZ01CQUFHalV6QlJNQjBHQTFVZApEZ1FXQkJROFZEbkpkNkhlcXVINk1xKzNHV3QxWG8xVUtqQWZCZ05WSFNNRUdEQVdnQlE4VkRuSmQ2SGVxdUg2Ck1xKzNHV3QxWG8xVUtqQVBCZ05WSFJNQkFmOEVCVEFEQVFIL01BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQ2wKd0NDRSttUlp4eWNBK0pIODduN05naitSQjA1ak5WTlpCb2UzQUNZRmwrL3NFbERMc3UzRmR3ZjRHOW91L0x2bQpDVVRTaHNFQmQ4YkZpWlVKN0dLSmloTHdESjRYS2lBckNHS0p4d3doVEtZUDlzaDAyUXNYV0kwc1orN1c4bll5ClJLM2Q0NFpySWczQXJDaFBXcEdqUmtWVy9YWTdTamxzWUxRSFNFR2tMLzlSeW1UWVIycGQ5czhGZXFremFkbHIKSm1jRUlFZFNtbXNjcDNQNEk3MW43bHFXRTIyVDluUkFMczlKK1hkMm9IbkhLazRkcTU1QXpsU3A4ODM3aU9FTwp2L25RZzFNUjJPOE9jcHAyU1JsYmlsbVlmeGRJaytLZXNZSmVDd1NiaUxucmNJeHdtdWJBZ0E3TjVWN1pFa1paCnVFY2c3TGV6cUJMaUVUUm1Ga01OCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
```

Comment:
These were just inlined versions of the files previously. To avoid the need to synchronize them, I changed the code to read the data from the file on init